packages/discovery-react-components/src/components/CIDocument/components/CIDocumentContent/CIDocumentContent.tsx

```diff
@@ -29,6 +29,7 @@ export interface CIDocumentContentProps {
   theme?: Theme;
   documentId?: string;
   onItemClick?: OnFieldClickFn;
+  highlightedIdsByColor?: { color: string; highlightLocationIds: string[] }[];
```

Comment:
Similar to `OnFieldClickFn`, consider making `{ color: string; highlightLocationIds: string[] }[]` into type. Developers reusing the component can use the type to create and manage a separate object.